.github/workflows/docker.yml

```diff
@@ -81,10 +81,9 @@ jobs:
           token: ${{ secrets.BOTPRESSOPS_PAT }}
       - uses: actions/checkout@v2
         with:
-          repository: botpress/actions
-          token: ${{ secrets.BOTPRESSOPS_PAT }}
-          path: "@actions"
-      - uses: ./@actions/build/docker
+          repository: botpress/gh-actions
+          path: "@gh-actions"
+      - uses: ./@gh-actions/build/docker
```

Comment:
Is it possible to use a certain tag version of the repo instead of the default branch (which is most likely master here)?

Reply:
@laurentlp it's possible to use commit hashes (we don't release tag versions for now) but we would need to keep it up to date or automate the process

Reply:
I created a v1 tag that matches the v1 branch not so long ago: https://github.com/botpress/gh-actions/releases/tag/v1

It contains your commit so I think you can use it here and if you ever need to update the action, you'll simply have to create a PR that points to the v1 branch.